save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProject.kt

```diff
@@ -89,14 +98,20 @@ class DownloadProject(val configProperties: ConfigProperties) {
         }
     }
 
-    private fun sendToBackendAndOrchestrator(id: Long, path: String) {
-        val execution = Execution(id, LocalDateTime.now(), LocalDateTime.now(), ExecutionStatus.PENDING, "1", path)
+    private fun sendToBackendAndOrchestrator(projectId: Long, path: String) {
+        val execution = Execution(projectId, LocalDateTime.now(), LocalDateTime.now(), ExecutionStatus.PENDING, "1", path)
         log.debug("Knock-Knock Backend")
         webClientBackend
             .post()
             .uri("/createExecution")
             .body(BodyInserters.fromValue(execution))
             .retrieve()
+            .onStatus({status -> status != HttpStatus.OK }) {
+                throw ResponseStatusException(
```

Comment:
Please add `log.error` here too, and specify status and message if they are available here

Reply:
Added